extension/public/popup.css

```diff
@@ -241,12 +242,14 @@ header {
 
   border-radius: 12px;
   background-color: #c54ccf;
+
```

Comment:
Please remove extra empty lines as it is not related to this commit. If there is a convention that you would like to follow, we can discuss it in a meeting and create a code formatting issue.

Reply:
I am following the convention that they left in this CSS file. They separated the different parts of the CSS file, though I have no idea why they do that,,,

Reply:
This new line is just above a closing bracket and not close to the change you made on line 236.